### PR TITLE
Adding post for CfST opening announcement

### PR DESCRIPTION
**Pull request by attacus**
Edited draft in Jekyll format for Call for Specialist Tracks opening announcement.

### CODE REVIEW

**Comment by glasnt** [comment]
:shipit:

File: _posts/2017-11-20-cfst-open.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+---
+layout: post
+title: Call for Specialist Tracks is now open!
+---
+
+
+Each year at PyCon AU, we dedicate a day to our Specialist Tracks. These are single-day “mini-conferences”, where each room's talks have a specific theme, curated by members of the community.
+
+
+In the past we have had tracks for Science and Data, Education, Internet of Things, Django, OpenStack, and more. If are a fan of any of the previous topics and would like to run the Track in 2018, we want to hear from you! If you have something new you want to try out, we want to hear from you!
```
Comment (attacus) [inline _posts/2017-11-20-cfst-open.md]: Missed a word here. "If you are a fan..."

Comment (glasnt) [reply _posts/2017-11-20-cfst-open.md]: ... after fixing that, :shipit:

**Comment by attacus** [comment]
Merged

**Pull request merged**